src/transformers/trainer.py

```diff
@@ -1567,6 +1563,9 @@ def log(self, logs: Dict[str, float]) -> None:
         if self.state.epoch is not None:
             logs["epoch"] = round(self.state.epoch, 2)
 
+        self.store_flos()
+        logs["total_flos"] = self.state.total_flos
```

Comment:
Can we leave it in the metrics as before? People have code relying on the logs and this change will break it.

Reply:
Done !